### PR TITLE
Developer 2716 - Update Fuse Quickstarts to version 6.2.1

### PR DESCRIPTION
**Pull request by Dantheman720**
Testing aweplug changes to make sure links aren't broken

### CODE REVIEW

File: _ext/pipeline.rb [hunk 1]
```diff
@@ -140,6 +140,30 @@
                                                           excludes: ['_fuse-quickstarts/template'],
```
Comment (paulrobinson) [inline _ext/pipeline.rb]: Do you need to ignore the directories that you are about to index below?

**Comment by paulrobinson** [comment]
@LightGuard can you review the code for this?

@Dantheman720, you'll need to go back to the Fuse person who requested this, for a review. before it's merged.

File: Gemfile.lock [hunk 2]
```diff
@@ -452,6 +453,3 @@ DEPENDENCIES
   uglifier (~> 2.0.1)
   uuid
   wraith (~> 1.3.0)
-
-BUNDLED WITH
```
Comment (LightGuard) [inline Gemfile.lock]: Are you using an old version of bundler?

File: _ext/pipeline.rb (hunk 1 above)
Comment (LightGuard) [inline _ext/pipeline.rb]: Ignore them for what, awestruct? The directories start with '_'.

**Comment by LightGuard** [comment]
Aside from my comments, looks okay.

**Comment by RubyGemTSL** [comment]
retest this please

**Comment by Dantheman720** [comment]
retest this please

**Comment by Dantheman720** [comment]
retest this please

File: _ext/pipeline.rb [hunk 3]
```diff
@@ -137,7 +137,8 @@
...
...switchyard/demos/policy-security-saml','_fuse-quickstarts/switchyard/demos/policy-security-wss-signencrypt','_fuse-quickstarts/switchyard/demos/policy-security-wss-username','_fuse-quickstarts/switchyard/demos/policy-transaction','_fuse-quickstarts/switchyard/demos/security-propagation','_fuse-quickstarts/switchyard/demos/transaction-propagation','_fuse-quickstarts/switchyard/demos/webapp-deploy','_fuse-quickstarts/switchyard/ear-deployment','_fuse-quickstarts/switchyard/http-binding','_fuse-quickstarts/switchyard/jca-inflow-activemq','_fuse-quickstarts/switchyard/jca-inflow-hornetq','_fuse-quickstarts/switchyard/jca-outbound-activemq','_fuse-quickstarts/switchyard/jca-outbound-hornetq','_fuse-quickstarts/switchyard/remote-invoker','_fuse-quickstarts/switchyard/rest-binding','_fuse-quickstarts/switchyard/soap-attachment','_fuse-quickstarts/switchyard/soap-binding-rpc','_fuse-quickstarts/switchyard/soap-mtom','_fuse-quickstarts/switchyard/transform-dozer','_fuse-quickstarts/switchyard/transform-jaxb','_fuse-quickstarts/switchyard/transform-json','_fuse-quickstarts/switchyard/transform-smooks','_fuse-quickstarts/switchyard/transform-xslt','_fuse-quickstarts/switchyard/validate-xml'],
```
Comment (paulrobinson) [inline _ext/pipeline.rb]: Can a regex be used here? This would save you from having to list every dir under 'switchyard' which may change in the future.

**Comment by Dantheman720** [comment]
This is ready for review.

**Comment by LightGuard** [comment]
@Dantheman720 This needs to be moved to the master of aweplug now, correct? Also who's the one to review this?

**Comment by Dantheman720** [comment]
Yeah, I will update the Gemfile for that.  Jonathan Antsey will need to review it once that happens.  I posted in the JIRA ticket about why we excluded switchyard.

**Comment by LightGuard** [comment]
@Dantheman720 Are we ready to retest?

**Comment by Dantheman720** [comment]
retest this please.

**Comment by Dantheman720** [comment]
retest this please

**Comment by RubyGemTSL** [comment]
retest this please

**Pull request merged**